style.css

```diff
@@ -79,3 +81,187 @@ header a {
   margin-right: 24px;
   color: #6070ff;
 }
+
+.project {
+  margin: 114px 24px 114px 24px;
+  display: grid;
+  gap: 88px;
+}
+
+.project-card {
+  box-sizing: border-box;
+  border-radius: 16px;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.pix {
+  height: 220px;
+  width: 100%;
+}
+
+.project-title {
+  font-family: "poppins", sans-serif;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: 12px;
+}
+
+.project-description {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  margin-top: 20px;
+  font-family: "poppins", sans-serif;
+}
+
+span {
+  color: #7a869a;
+  font-family: "poppins", sans-serif;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.project-content {
+  font-size: 15px;
+  line-height: 24px;
+  font-family: "poppins", sans-serif;
+  color: #344563;
+  margin-top: 20px;
+}
+
+.lang {
+  display: flex;
+  gap: 8px;
+  margin-top: 12px;
+}
+
+.btn {
+  border-radius: 8px;
+  font-weight: 500;
+  background-color: #ebebff;
+  font-size: 12px;
+  color: #6070ff;
+  line-height: 16px;
+  font-family: "poppins", sans-serif;
+  padding: 4px 12px;
+}
+
+.bon {
+  margin-top: 14px;
+  font-weight: 500;
+  line-height: 24px;
+  font-family: "poppins", sans-serif;
+  border-radius: 8px;
+  border: 1px solid #7f8cff;
+  font-size: 17px;
+  background-color: #fff;
+  box-sizing: border-box;
+  padding: 12px;
+  color: #396df2;
+}
```

Comment:
- [ ] The `See Project` `button` should have different display properties in different states (hover, enabled, disabled). Please apply this change.